ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-rollback-nightly.yaml

```diff
@@ -106,8 +106,35 @@ tests:
           memory: 200Mi
     - ref: cucushift-chainupgrade-toimage
     workflow: cucushift-installer-rehearse-azure-upi
-- as: gcp-ipi-ovn-ipsec-f14
-  cron: 19 6 6,22 * *
+- as: baremetal-ipi-ovn-ipv4-f360
+  cluster: build05
+  cron: 20 5 12 * *
+  steps:
+    cluster_profile: equinix-ocp-metal-qe
+    env:
+      AUX_HOST: openshift-qe-metal-ci.arm.eng.rdu2.redhat.com
+      DISCONNECTED: "false"
+      RESERVE_BOOTSTRAP: "false"
+      architecture: amd64
```

Comment:
What's the architecture? ARM or AMD? openshift-qe-metal-ci.`arm`g.rdu2.redhat.com?  cc @zniu1011

Reply:
I copied from e2e jobs.